helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -719,6 +736,17 @@ public void cleanupLiveInstanceOwners() {
       clientMap.clear();
     }
     _liveInstanceOwners.clear();
+
+    boolean status = false;
+    try {
+      status = ThreadLeakageChecker.afterClassCheck(testClassName);
+    } catch (Exception e) {
+      System.out.println("ThreadLeakageChecker exception:" + e.getStackTrace());
```

Comment:
Why system out print instead of log? If I redirect the test log into a file for debugging, I won't see this information.

Reply:
Good point. changed to log.error